iroh-willow/src/engine/actor.rs

```diff
@@ -480,6 +531,44 @@ impl<S: Storage> Actor<S> {
                 let res = self.store.auth().resolve_interests(interests);
                 send_reply(reply, res.map_err(anyhow::Error::from))
             }
+            Input::SubscribeArea {
+                namespace,
+                area,
+                params,
+                sender,
+            } => {
+                let store = self.store.clone();
+                self.tasks.spawn_local(async move {
+                    let mut stream = store.entries().subscribe_area(namespace, area, params);
+                    while let Some(event) = stream.next().await {
+                        if sender.send(event).await.is_err() {
+                            break;
+                        }
+                    }
+                });
```

Comment:
This kind of forwarding of events seems slightly weird to me. Although I guess it's either this or more invasive changes to the `trait Storage`/`trait EntryStorage` (allowing you to just give it a sender instead of it returning a stream).

Reply:
I agree that this is not ideal. We do similar things already currently in the RPC boundary for old iroh-docs.
Maybe it makes sense for the subscribe methods on the store trait to take in a sender instead of returning a stream.
This brings up the question though which trait it should be generic over: Sink? ufotofu::Consumer? a custom trait that has only `async fn send(item: Event)`?

Reply:
Added a comment for now.